packages/langium/src/default-module.ts

```diff
@@ -62,7 +62,7 @@ export function createDefaultModule(context: DefaultModuleContext): Module<Langi
                 RuleInterpreter: () => new RuleInterpreter()
             },
             DocumentSymbolProvider: (services) => new DefaultDocumentSymbolProvider(services),
-            HoverProvider: (services) => new MultilineCommentHoverProvider(services),
+            HoverProvider: (services) => new LangiumGrammarHoverProvider(services),
```

Comment:
Suggestion: We shouldn't use the `LangiumGrammarHoverProvider` as the default hover provider for all services, but only for the langium grammar service. See:

https://github.com/langium/langium/blob/244442e047424c3c519d1017a81c97225e4545ae/packages/langium/src/grammar/langium-grammar-module.ts#L30-L34

Reply:
Wrong module...